components/header/index.js

```diff
@@ -1,22 +1,20 @@
-import colors from '../../color/color.json';
+import styles from './header.module.css'
```

Comment:
NIT : the better name here can be 
import classNames from './header.module.css'